zerver/migrations/0321_realm_creation_log_backfill.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 3.1.8 on 2021-04-20 10:09
+
+from django.db import migrations
+from django.db.backends.postgresql.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+def backfill_realm_creation_log_events(
+    apps: StateApps, schema_editor: DatabaseSchemaEditor
+) -> None:
+    RealmAuditLog = apps.get_model("zerver", "RealmAuditLog")
+    Realm = apps.get_model("zerver", "Realm")
+
+    objects_to_create = []
+    for realm in Realm.objects.all():
+        entry = RealmAuditLog(
+            realm=realm, event_type=215, event_time=realm.date_created, backfilled=True
```

Comment:
For migrations like this, it's best to write `REALM_CREATED`= 215` and then use the variable, so `grep` finds uses of the code; @mateuszmandera can point to the technique.